sql/parser/expr.go

```diff
@@ -415,12 +428,7 @@ LOOP:
 	return vPath, nil
 }
 
-func (p *Parser) parseExprList(leftToken, rightToken scanner.Token) (expr.LiteralExprList, error) {
-	// Parse ( or [ token.
-	if tok, pos, lit := p.ScanIgnoreWhitespace(); tok != leftToken {
-		return nil, newParseError(scanner.Tokstr(tok, lit), []string{leftToken.String()}, pos)
-	}
-
+func (p *Parser) parseListUntil(rightToken scanner.Token) (expr.LiteralExprList, error) {
```

Comment:
Good idea!
I'd like to be pedantic here and rename this to `parseExprListUntil` for consistency